lib/handlers/allow.js

```diff
@@ -27,9 +27,11 @@ function allow (mode) {
       suffix: ldp.suffixAcl,
       strictOrigin: ldp.strictOrigin
     })
+    req.acl = acl
 
     getUserId(req, function (err, userId) {
       if (err) return next(err)
+      req.userId = userId
```

Comment:
As we discussed, not in all cases (could be in `req.session.userId` or `locals.oidc.webIdFromClaims(req.claims)`). Opened #517 for this.